docs/unit-testing-for-monitoring-metrics.md

```diff
@@ -84,12 +98,17 @@ socat -v UNIX-LISTEN:$SOCKETFILE,fork TCP-CONNECT:127.0.0.1:12345
 For debugging it might be useful to run these commands manually in three different terminals to see where the connection might break down. The docker log file should show if data flows through or not, too.
 
 
-### Stopping the Network script
+#### Stopping the Network script
 
 Stop the script with Ctrl-C. The script then will do its best to stop the ssh and socal processes it spawned.
 
+### The UDP way
+
+If required you can adapt the [deploy script for the Unix domain socket way](../hack/local-monitor-testing/deploy_MDM_VM.sh) deploys such a VM called $USER-mdm-link on Azure, configures it and installs the mdm container.
```

Comment:
what would make it required to adapt/change the script? what are the firewall rules mentioned below that need to be in place?